src/Components/Form/Form.tsx

```diff
@@ -0,0 +1,47 @@
+import { Container, Button } from '@mui/material'
+import { useState } from 'react'
+import { NumberInputTextField } from '../NumberInputTextField/NumberInputTextField'
+
+export function Form(): JSX.Element {
+    const [focusTime, setFocusTime] = useState<number>(25)
+    const [brakeTime, setBrakeTime] = useState<number>(5)
+    const [longerBrakeTime, setLongerBrakeTime] = useState<number>(15)
+    return (
+        <>
+            <br />
+            <Container style={{ display: 'flex' }}>
+                <NumberInputTextField
+                    label="Focus time"
+                    value={focusTime}
+                    onChangeEvent={(e) => {
```

Comment:
Those `onChangeEvent` props look pretty similar, you can avoid repetition this way for example:
```
    const handleChange =
        (cb: (number: number) => void) =>
        (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
            event.preventDefault()
            const focusValue = parseInt(event.target.value, 10)
            cb(focusValue)
        }
        
{...}
      
<NumberInputTextField
    label="Focus time"
    value={focusTime}
    onChangeEvent={handleChange(setFocusTime)}
/>
<NumberInputTextField
    label="Brake time"
    value={brakeTime}
    onChangeEvent={handleChange(setBrakeTime)}
/>
<NumberInputTextField
    label="Longer brake time"
    value={longerBrakeTime}
    onChangeEvent={handleChange(setLongerBrakeTime)}
/>
```